src/python/platforms/fuchsia/device.py

```diff
@@ -35,6 +36,14 @@
 _QEMU_WAIT_SECONDS = 30
 
 
+class QemuRunner(new_process.ProcessRunner):
+
+  def get_command(self, additional_args=None):
+    import pipes
```

Comment:
nit: I know this is from my diff, but please do this properly and move this do the top like so, taking into account Python 2/3 differences:

```python
try:
  from shlex import quote
except ImportError:
  from pipes import quote
```

Reply:
[No longer necessary as this code has been removed]